agent/bench-scripts/tests/pbench-fio/test-20.txt

```diff
@@ -220,27 +220,27 @@ fio job complete
-/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/1-read-4KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/1-read-4KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,8765 > client-result.txt 2>&1'
-/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/2-read-64KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/2-read-64KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,8765 > client-result.txt 2>&1'
-/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/3-read-1024KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/3-read-1024KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,8765 > client-result.txt 2>&1'
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/1-read-4KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/1-read-4KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,42 > client-result.txt 2>&1'
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/2-read-64KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/2-read-64KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,42 > client-result.txt 2>&1'
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/ssh -o BatchMode=yes -o StrictHostKeyChecking=no abc killall fio >/dev/null 2>&1; mkdir -p /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/3-read-1024KiB/sample1 && cd /var/tmp/pbench-test-bench/pbench-agent/fio_test-20_1900.01.01T00.00.00/3-read-1024KiB/sample1 && screen -dmS fio-server bash -c '/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/bm --server=,42 > client-result.txt 2>&1'
```

Comment:
Just to confirm, is this the sequencing that we expect?  (We do each of the three iterations on the first server; then we do each of them on the second; etc....I would naively have thought that we would hit each server on each iteration before advancing to the next iteration.)

Reply:
Yes:  the output is pre-sorted.